src/app/models/propTypes/inline/index.js

```diff
@@ -5,7 +5,6 @@ import { inlineLinkBlockPropTypes } from '../inlineLink';
 
 export const inlineModelPropTypes = {
   language: string.isRequired,
-  text: string.isRequired,
```

Comment:
I think text is right here, e.g. this model https://github.com/bbc/simorgh/blob/ReactWarning3289/data/persian/articles/c19v65z3lg5o.json#L104-L116